module/metrics/verification.go

```diff
@@ -348,3 +375,15 @@ func (vc *VerificationCollector) OnChunkDataPackArrivedAtFetcher() {
 func (vc *VerificationCollector) OnVerifiableChunkSentToVerifier() {
 	vc.sentVerifiableChunksTotal.Inc()
 }
+
+// OnChunkConsumerJobDone is invoked by chunk consumer whenever it is notified a job is done by a worker. It
+// sets the last processed chunk job index.
+func (vc *VerificationCollector) OnChunkConsumerJobDone(processedIndex uint64) {
+	vc.lastProcessedChunkJobIndexChunkConsumer.Set(float64(processedIndex))
+}
+
+// OnBlockConsumerJobDone is invoked by block consumer whenever it is notified a job is done by a worker. It
+// sets the last processed block job index.
+func (vc *VerificationCollector) OnBlockConsumerJobDone(processedIndex uint64) {
+	vc.lastProcessedBlockJobIndexBlockConsumer.Set(float64(processedIndex))
```

Comment:
The only concern is that the processedIndex might go backwards in a short period of time. 

For instance, if 3 workers are working on job index: 6, 7, 8. And they finish in this order: `8, 6, 7`, then in the end, the processed index would be `7`.

Reply:
A job is marked as _processed_ only when _all jobs_ preceding it are _processed_:
https://github.com/onflow/flow-go/blob/71e9ff7092b8a20ff71265b35b55101d842b7d8e/module/jobqueue/consumer.go#L312-L314

So the `processedIndex` is a _monotonically increasing_ index. Since`lastProcessedBlockJobIndexBlockConsumer` gauges the `processedIndex`, it always reflects the last processed job in a monotonically increasing way, without any backward behavior. 

 > For instance, if 3 workers are working on job index: 6, 7, 8. And they finish in this order: 8, 6, 7, then in the end, the metrics for processed index would be shown as 7, but the actual processed index is 8.
 
 In this example, the `processedIndex = 5` before jobs 6, 7, and 8 start. Having them finish in the order of 8, 6, and 7: 
 - The `processedIndex` does not move after 8 finishes, since 6 and 7 are not yet _processed_. 
 - The `processedIndex` increments from 5 -> 6 after 6 finishes, since all jobs preceding `6` are already processed. 
 - The `processedIndex` moves forward from 6-> 7 -> 8 after 7 is done since all jobs preceding `8` are already processed. 
 
 In all cases, the `processedIndex` reflects the _last processed job_, which is the point from which the job consumer picks up upon a restart.